pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -129,7 +130,8 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) (*apps.Deployment, error)
 		},
 		Spec: apps.DeploymentSpec{
 			Selector: &metav1.LabelSelector{
-				MatchLabels: c.getPodLabels(mgrConfig, false),
+				MatchLabels:          c.getPodLabels(mgrConfig, false),
+				RevisionHistoryLimit: opcontroller.RevisionHistoryLimit(),
```

Comment:
would this be an env value from rook operator pod?

Reply:
@parth-gr  wrote:
> would this be an env value from rook operator pod?

I am not quite sure I understand. I added this via a operator config variable which can be set via configmap or env variable. does this answer your question?

Reply:
Yaa it can be added at both places
https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml#L618
https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml#L24
The change looks still left to do

Reply:
JUst a remark, Env variable have more precedence than the config map values

Reply:
@parth-gr  wrote:
> Yaa it can be added at both places
> https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml#L618
> https://github.com/rook/rook/blob/master/deploy/examples/operator.yaml#L24 
> The change looks still left to do

do you just want to test some example in the ci?

Reply:
Added, fwiw. :)